src/test/resources/mappings/twoDayTimeEntryDeleteAction.json

```diff
@@ -164,10 +166,12 @@
               "agreementId": "c0a80193-87a3-1ff0-8187-a3bfe2b80004",
               "accrualDate": "2023-04-23",
               "accrualTypeId": "5f06e6ce-1422-4a0c-89dd-f4952e735202",
-              "cumulativeTotal": 1000,
+              "cumulativeTotal": 1420,
               "cumulativeTarget": 1200,
               "contributions": {
-                "timeEntries": {},
```

Comment:
Just a thought, Would it be worth having another time entry 23rd / 24th just to make sure the delete operation only deletes the relevant time entry Id's?

Reply:
Added some extra time entries to test this